lib/new_relic/agent/monitors/synthetics_monitor.rb

```diff
@@ -6,6 +6,7 @@ module NewRelic
   module Agent
     class SyntheticsMonitor < InboundRequestMonitor
       SYNTHETICS_HEADER_KEY = 'HTTP_X_NEWRELIC_SYNTHETICS'.freeze
+      SYNTHETICS_INFO_HEADER_KEY = 'HTTP_X_NEWRELIC_SYNTHETICS_INFO'
```

Comment:
Just for uniformity, I think it'd be good to remove the `#freeze` call on the other constant's value.

Reply:
updated in https://github.com/newrelic/newrelic-ruby-agent/pull/2203/commits/705bcd9fe0c9a991c3bb3638491020a704d2b3ae